spec/exchanges/huobi/integration/market_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+RSpec.describe 'Huobi integration specs' do
+  let(:client) { Cryptoexchange::Client.new }
+
+  it 'fetch pairs' do
+    pairs = client.pairs('huobi')
+    expect(pairs).not_to be_empty
+
+    pair = pairs.first
+    expect(pair.base).not_to be_nil
+    expect(pair.target).not_to be_nil
+    expect(pair.market).to eq 'huobi'
+  end
+
+  it 'fetch ticker' do
+    eth_cny_pair = Cryptoexchange::Models::MarketPair.new(base: 'ETH', target: 'CNY', market: 'huobi')
+    ticker = client.ticker(eth_cny_pair)
+
+    expect(ticker.base).to eq 'ETH'
+    expect(ticker.target).to eq 'CNY'
+    expect(ticker.market).to eq 'huobi'
+    expect(ticker.last).to be_a Numeric
+    expect(ticker.bid).to be_a Numeric
+    expect(ticker.ask).to be_a Numeric
+    expect(ticker.high).to be_a Numeric
+    expect(ticker.volume).to be_a Numeric
+    expect(ticker.timestamp).to be_a Numeric
+    expect(DateTime.strptime(ticker.timestamp.to_s, '%s').year).to be <= 5000
```

Comment:
@yihangho @Swingcloud 
Regardless of whether we should use our own timestamp, let's use `Time.zone.now.year` to test this instead of `5000`

Reply:
That's fine by me

Reply:
Should I update this PR or will it be part of the global replace?

Reply:
@yihangho just change yours